src/pineko/evolve.py

```diff
@@ -426,15 +426,15 @@ def prepare(operator):
         fktable = grid.evolve_with_slice_iter2(
             prepare(operators1),
             prepare(operators2),
-            ren1=mur2_grid,
+            ren1=xir*xir*mur2_grid,
```

Comment:
This looks correct, I was introduced in #181, my bad

Reply:
agreed:
https://github.com/NNPDF/pineko/blob/29a02b95774cd3b39af1334ca2e3b30dbd83b473/src/pineko/evolve.py#L278